### ISSUE TITLE
RSS Aggregator

### ISSUE THREAD

**Issue by pipedream-component-development** [PipedreamHQ/pipedream#12167]
### App
RSS Aggregator

### Summary:
RSS triggers hang

### Details:
These RSS triggers hang, problem with pulling the data.
In the "Select Event" section of a trigger I get a message "Action Required, In RSS, generate test events and then search for new events." and in the corresponding field where the data should appear I have a constant "Waiting for events..." message.
And nothing happens. I cannot select an event to test.

### Screenshots:
https://www.youtube.com/watch?v=t7QSm4WUvJ0
![Screenshot](https://storage.tally.so/private/rss-workflow.png?id=EdZdg2&accessToken=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6IkVkWmRnMiIsImZvcm1JZCI6Im5yampSdiIsImlhdCI6MTcxNzEwMDgxN30.18sr-I24ox_YyWQ6bwGoypSqe9sExa8aMMPp2t5okko&signature=40fb54189d607e2bc58cf0d3b3f099462b708eb06d398015f60738ec7360f1f9)

**Comment by slishnevsky** [comment]
Hmmm.
Very interesting.
It seems that some RSS triggers got "disabled" when others got "enabled".
By "disabled" / "enabled" I mean "capable of retrieving data".

I have five RSS triggers in one workflow.
Previously, only two out of five were working.
After a few days, a third began to work, but the last two - didn't.

Today I noticed that one of those that I reported began to work while another didn't.
When I clicked on "Try Now" for the first RSS trigger, it began to work, but the second stopped.
When I clicked on "Try Now" for the second RSS trigger, it began to work, but the first stopped.

And each time I click "Try now", one RSS trigger begins to work, while the other one stops.

Not sure if this is by design or a bug.
Thanks.
